frontend/alert/src/components/ManageAlert.module.css

```diff
@@ -0,0 +1,29 @@
+.actionbutton > * {
```

Comment:
are these rules that can't be done in StyledComponents? wondering the reasoning behind this bit of CSS modules.

Reply:
My bad - did not know you could have these in styled components! Fixed.